packages/asc-ui/src/components/Paragraph/ParagraphStyle.ts

```diff
@@ -17,14 +17,6 @@ export const ParagraphStyleCSS = css<Props>`
       font-weight: 700;
```

Comment:
Please do remove this strong prop here, as we already use it in typography

Reply:
done